sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -4903,7 +5021,8 @@ private void SetNameFieldsIfNull()
             BlobRequestConditions conditions,
             bool async,
             RequestContext context,
-            string operationName = default)
+            string operationName = default,
+            bool? userPrincipalName = default)
```

Comment:
We should move this up above the `bool async`.  Everything below `async` is about calling conventions.